rdtools/degradation.py

```diff
@@ -222,7 +244,7 @@ def remove_outliers(x):
   
     med1 = np.median(YoY_filtered1)                       
     
-    # bootstrap to determine 95% CI for the 2 different outlier removal methods
+    # bootstrap to determine 68% CI for the 2 different outlier removal methods
```

Comment:
This is a questions for @dirkjordan and @cdeline , but my guess is that this aligns with the standard error one report from a regression. (I just updated the comment to reflect the code.)